certified-connectors/personr/personr.swagger.json

```diff
@@ -0,0 +1,491 @@
+                    "nameLast": {
+                      "type": "string",
+                      "description": "nameLast"
+                    },
+                    "flowName": {
+                      "type": "string",
+                      "description": "flowName"
+                    },
+                    "applicantStatus": {
+                      "type": "string",
+                      "description": "applicantStatus"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api-verificationlink-create": {
+      "post": {
+        "summary": "Create Verification Link",
+        "description": "After you have created an applicant, you can use this action to create a verification link. The verification link will take the applicant to a page where they will be able to provide required documents and complete the verification.",
+        "operationId": "Api-verificationlink-create",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "applicant": {
+                  "type": "string",
+                  "description": "applicant",
+                  "title": "Applicant ID"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "response": {
+                  "type": "object",
+                  "properties": {
+                    "verificationLink": {
+                      "type": "string",
+                      "description": "verificationLink"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api-document-upload": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Upload Documents to an Applicant",
+        "operationId": "Api-document-upload",
+        "description": "Use this action to upload documents to an applicant, instead of sending them a link. You can only send one document at a time, when all required documents are uploaded the applicant will move to the pending status.  If you're uploading double-side documents, make sure to send FRONT_SIDE and BACK_SIDE in the idDocSubType parameter. When both sides are present and valid the verification step will be considered complete. Images must be uploaded in base64 format.",
```

Comment:
Hello @charliewesterman, 

please keep all operations descriptions short to fit in the text box. please see our [docs](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).